diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/files/NewlinesRule.kt

```diff
@@ -407,33 +415,59 @@ class NewlinesRule(private val configRules: List<RulesConfig>) : Rule("newlines"
         .takeWhile { it.elementType in expressionTypes }
         .singleOrNull()
```

Comment:
I see a problem with this method now: it should return true, if statement is a single dot qualified expression on one line, but you wanted it to return `true` for property accesses too, I assume. I think it'll make more sense to check if list of parents here has no more that 1 element and return `true` in this case and `false` otherwise.